docs/user-docs/latest/plugins/introduction-to-plugins.md

```diff
@@ -6,15 +6,31 @@ canonicalUrl: https://docs.strapi.io/user-docs/latest/plugins/introduction-to-pl
 
 # Introduction to plugins
 
-Strapi is built around different types of plugins. Core plugins are essential for your Strapi application to function, and therefore cannot be deactivated. Pre-installed plugins, such as Internationalization, are installed in a default Strapi application. Additional plugins that extend and customize Strapi applications are available in the [Marketplace](../plugins/installing-plugins-via-marketplace.md).
+Strapi is built around different types of plugins. Every default Strapi application comes with the following pre-installed plugins:
 
-::: note
-Core plugins which include the Content Manager and the Content-type Builder are documented in their own sections of the user guide as they are quite specific (see [Introduction to the Content Manager](../content-manager/introduction-to-content-manager.md) and [Introduction to the Content-type Builder](../content-types-builder/introduction-to-content-types-builder.md)). This Plugins section focuses on how to manage plugins in general and provides documentation for the other, non-core plugins.
-:::
+* Content Manager (see [Introduction to the Content Manager](../content-manager/introduction-to-content-manager.md))
+* Content Type Builder (see [Introduction to the Content-type Builder](../content-types-builder/introduction-to-content-types-builder.md))
+* Email
+* Media Library (implemented via the [Upload plugin](../../../developer-docs/latest/plugins/upload.html))
+* Internationalization
+* Roles and Permissions
```

Comment:
I don't think it's the case right this moment but should check with engineering when we plan to stop installing U&P by default.

Reply:
![image](https://user-images.githubusercontent.com/8593673/172386370-c06aeecc-7cc4-4d0f-a724-be483571963d.png)

Reply:
Guess for now it's fine :)